GliaWidgets/SecureConversations/Welcome/Theme+SecureConversationsWelcome.swift

```diff
@@ -183,7 +183,7 @@ extension Theme {
                 errorIcon: Asset.uploadError.image,
                 errorIconColor: color.systemNegative,
                 backgroundColor: color.primary,
-                errorBackgroundColor: Color.lightGrey,
+                errorBackgroundColor: Color.baseNeutral,
```

Comment:
Do we need to use color from palette?
```suggestion
                errorBackgroundColor: color.baseNeutral,
```

Reply:
Why didn't we use it in the first place?

Reply:
I think we did not use it by mistake. We need to use ThemeColor instead of constants, because the first one might be changed using GlobalColors. The idea is to not use `Color` constant at all, only as default value for ThemeColor

Reply:
fixed [f8b8b60](https://github.com/salemove/ios-sdk-widgets/pull/777/commits/f8b8b600ed2c3cccef497d0c35a59dee80a4cfea)